deploy/0.9.0/contracts.json

```diff
@@ -0,0 +1,3 @@
+{
+    "MerkleBox": "0x693c78cA4C7B061EadF65AB3e6c12448Af2b6Cfb"
```

Comment:
Please use this format, even if only one network: https://github.com/bloq/sol-address-list/blob/master/deploy/0.9.9/contracts.json

The above ad hoc format does not give us deployment version or network.